HW_4/task_2.py

```diff
@@ -0,0 +1,22 @@
+"""
+Задайте список случайных чисел. Выведите список чисел, которые не повторяются в заданном списке.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
+[2, 4, 6, 8]
+"""
+
+from random import randint
+
+n = int(input("Введите натуральное число N "))
+my_list = [randint(0, n) for i in range(n)]
+print(my_list)
+
+result = []
+for number in my_list:
+    if not my_list.count(number) > 1:

```

Comment:
Абсолютно верно, хотя условие немного своеобразное. Я не говорю, что это неправильно, но почему бы не сделать так?
```suggestion
    if my_list.count(number) == 1:
```